gradle/verification-metadata.xml

```diff
@@ -551,9 +551,9 @@
             <sha256 value="e5f60ab215d43f2614872bac5fdf9aaae3fd734bd67e93bf11a40007ec78d3f2" origin="Artifact is not signed"/>
          </artifact>
       </component>
-      <component group="com.gradle.publish" name="plugin-publish-plugin" version="0.16.0">
-         <artifact name="plugin-publish-plugin-0.16.0.jar">
-            <sha256 value="1a18d90631c6d20702453da255c7187b00de3a07efa4c41e22bf6385ccbf42d4" origin="Artifact is not signed"/>
+      <component group="com.gradle.publish" name="plugin-publish-plugin" version="0.17.0">
+         <artifact name="plugin-publish-plugin-0.17.0.jar">
+            <sha256 value="7a6033b0a8fdb922be165f8997bcb21a637e2f1b078e4c8435ee3f88c2754831" origin="Generated by Gradle"/>
```

Comment:
Can you please edit the origin to say this is there because it is not signed?

Reply:
Sure, done.